addons/chemical/stringtable.xml

```diff
@@ -426,7 +426,7 @@
         </Key>
         <Key ID="STR_kat_chemical_LvL1_Gas">
             <English>Toxic gas (Level 1)</English>
-            <German>Toxic gas (Level 1)</German>
+            <German>Giftgas (Level 1)</German>
```

Comment:
Pls change Giftgas to Senfgas since i made the mistake and did not change every toxic gas to senfgas

Reply:
Sorry, i changed it.